angular/projects/admin-nrpti/src/app/explore-panel/explore-panel.component.html

```diff
@@ -147,7 +148,7 @@
         </div>
       </div>
       <div class="col-4">
-        <label for="sourceSystemFilter">Source System</label>
+        <!-- <label for="sourceSystemFilter">Source System</label>
```

Comment:
Probably can remove the commented out stuff, uness it's being kept for re-adding or reference?

Reply:
bafd6120d732c2dc459f782faee36eaa19dbd513